### PR TITLE
Russian translation update

### PR DESCRIPTION
**Pull request by Valey**
Russian translation update:

    update strings.xml
    add sura_name.xml
    add readers.xml

### CODE REVIEW

**Comment by Valey** [comment]
Russian translation update:
 - update strings.xml
 - add sura_name.xml
 - add readers.xml

File: app/src/main/res/values-ru/sura_names.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string-array name="sura_names">
+    <item>аль-Фатиха (Открывающая Книгу)</item>
```
Comment (ozbek) [inline app/src/main/res/values-ru/sura_names.xml]: Where did you get the Sura Names from?
Normally, prefixes are capitalized, but your version are all with lower case.

Comment (Valey) [inline app/src/main/res/values-ru/sura_names.xml]: I verified with 2 translation. In the first one all letters was capitalized, but in the second one prefixes was lowercase. I'll check it again

Comment (ozbek) [inline app/src/main/res/values-ru/sura_names.xml]: Thanks. 
Just to be clear, I wanted to say _the first letters of prefixes are capitalized_ (not whole prefix); i.e. Аль-Фатиха (not АЛЬ-Фатиха).

**Comment by Valey** [comment]
Yes, I understood. My imam said that it doesn't matter, so I changed prefixes to capitalized.

**Comment by ahmedre** [comment]
jazakAllah khairan!

**Pull request merged**